src/meshapi/models/member.py

```diff
@@ -4,8 +4,21 @@
 from django.db.models.fields import EmailField
 from django_jsonform.models.fields import ArrayField as JSONFormArrayField
 
+class ContactPreference(models.TextChoices):
+    EMAIL = 'EMAIL', 'Email'
+    PHONE = 'PHONE', 'Phone'
+    SLACK = 'SLACK', 'Slack'
+    OTHER = 'OTHER', 'Other'
 
 class Member(models.Model):
+
+    contact_preference = models.CharField(
+        max_length=10,
```

Comment:
What does setting this parameter do for us? I don't think we're setting it anywhere else (for example `LinkType` on the `Link` model). Does this help with database size or something?